common/spec/dependabot/pull_request_creator_spec.rb

```diff
@@ -45,6 +46,7 @@
   let(:source) { Dependabot::Source.new(provider: "github", repo: "gc/bump") }
   let(:files) { [gemfile] }
   let(:base_commit) { "basecommitsha" }
+  let(:provider_metadata) { { work_item: 123 } }
```

Comment:
Maybe better to set this to nil here and only set it with a `work_item` in the `with an Azure source` context
```suggestion
  let(:provider_metadata) { nil }
```